File: eform-client/src/app/models/advanced/adv-entity-selectable-group-list-request.model.ts
export class AdvEntitySelectableGroupListRequestModel {
  public sort: string;
  nameFilter: string;
  pageIndex: number;
  pageSize: number;
  isSortDsc: boolean;
  offset: number;

  constructor() {
    this.sort = '';
    this.nameFilter = '';
    this.pageSize = 10;
    this.pageIndex = 0;
    this.offset = 0;
  }
}

File: eFormAPI/eFormAPI.Web/Infrastructure/Models/SelectableList/AdvEntitySelectableGroupListRequestModel.cs
namespace Microting.eFormApi.BasePn.Infrastructure.Models.SelectableList
{
    public class AdvEntitySelectableGroupListRequestModel
    {
        public string Sort { get; set; }
        public string NameFilter { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public bool IsSortDsc { get; set; }
    }
}